docs/dev/applications/h2network.md

```diff
@@ -6,6 +6,31 @@ prev_section: system
 permalink: /docs/dev/h2network/
 ---
 
+
+
+`H2Network` extends the `H2` / `H2GIS` geospatial database to provide graph routing functionalities. 
+
+`H2Network` is based on the [Java Network Analyzer](https://github.com/orbisgis/java-network-analyzer) (`JNA`) library. 
+
+JNA provides a collection of graph theory and social network analysis algorithms. These algorithms are implemented on mathematical graphs using the [JGraphT](https://jgrapht.org/) library.
+
+### Install H2Network
+
+To install `H2Network` execute the two following instructions.
+
+{% highlight mysql %}
+CREATE ALIAS IF NOT EXISTS H2GIS_NETWORK FOR "org.h2gis.network.functions.NetworkFunctions.load";
+CALL H2GIS_NETWORK();
+{% endhighlight %}
+
+
+### H2Network functions
+
 The following H2Network functions are available:
```

Comment:
The following SQL functions are available:

Reply:
ok